websites/S/Soundgasm/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({
+		clientId: "1195258708368822312",
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				play: "general.playing",
+				pause: "general.paused",
+				upload: "youtube.upload",
+			},
+			await presence.getSetting<string>("lang").catch(() => "en")
+		);
+	},
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/QIxjVEX.png",
+}
+
+let data,
+	audioTimestamps,
+	audioElement,
+	strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		newLang = await presence.getSetting<string>("ID").catch(() => "en");
```

Comment:
You do not have a `multiLang` setting named `ID` or `lang` in your `metadata.json` file.